### DISCUSSION TITLE
Many to many relation filter with _ne

### DISCUSSION THREAD

**Discussion by landorid** [strapi/strapi#8390] [Legacy Topics]
I have 2 collections called `Post` and `User`. In the `Post` collection i have a `done_by` many-to-many relation to `User`.

If a user marked a post as done I save it under `done_by`. 
I can filter done post like this:
` ctx.query._where.done_by_user = userId;`

But when i try to do this:
` ctx.query._where.done_by_user_ne = userId;`
to filter post not marked with done, the query doesn't work. (It works only if the post was marked only with one user.)

How to filter entries with no `done_by_user` relation?

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
